axelrod/tournament.py

```diff
@@ -25,10 +25,21 @@ def __init__(self, players, name='axelrod', game=None, turns=200,
             repetitions=repetitions)
         self.deterministic_cache = {}
         self.noise = noise
-
         self._processes = processes
         self._logger = logging.getLogger(__name__)
 
+    @property
+    def players(self):
+        return self._players
+
+    @players.setter
+    def players(self, players):
+        newplayers = []
+        for player in players:
+            player.turns = self.turns
```

Comment:
Small suggestion: call this `player.tournament_length` or something like that? `player.turns` could be understood as a counter of how many turns have happened so far or something like that?